Explorer/Assets/DCL/PluginSystem/Global/DynamicSettings.cs

```diff
@@ -0,0 +1,19 @@
+using DCL.AssetsProvision;
+using DCL.Character;
+using DCL.PluginSystem;
+using Diagnostics.ReportsHandling;
+using ECS.Prioritization;
+using System;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
+
+namespace DCL.PluginSystem.Global
+{
+    [Serializable]
+    public class DynamicSettings : IDCLPluginSettings
+    {
+        [field: SerializeField]
+        public AssetReferenceGameObject PopupCloserView { get; private set; }

```

Comment:
It's ok but in order to guaruntee that this game object contains a certain component it's better to use the derived class from `ComponentReference<TComponent>`

Reply:
ComponentReference is not serializable tho, right?